pkg/ingress/query_test.go

```diff
@@ -120,12 +120,142 @@ func TestQueryIngressLabels(t *testing.T) {
 	assert.Equal(t, "false", dep.Labels["another"])
 }
 
+func TestQueryIngressWithHosts(t *testing.T) {
+	enabled := true
+	jaeger := v1.NewJaeger(types.NamespacedName{Name: "TestQueryIngressWithHosts"})
+	jaeger.Spec.Ingress.Enabled = &enabled
+	jaeger.Spec.Ingress.Hosts = []string{"test-host-1"}
+
+	ingress := NewQueryIngress(jaeger)
+
+	dep := ingress.Get()
+
+	assert.NotNil(t, dep)
+	assert.Nil(t, dep.Spec.Backend)
+	assert.Len(t, dep.Spec.Rules, 1)
+	assert.Empty(t, dep.Spec.Rules[0].HTTP.Paths[0].Path)
```

Comment:
Shouldn't `Backend` also be tested here and in the next test?

Reply:
@objectiser you are right, added the missing assertions